sematic/examples/cifar_classifier/main.py

```diff
@@ -0,0 +1,107 @@
+# Standard Library
+import argparse
+import logging
+from dataclasses import replace
+
+# Sematic
+import sematic
+from sematic.ee.ray import RayNodeConfig
+from sematic.examples.cifar_classifier.pipeline import pipeline
+from sematic.examples.cifar_classifier.train_eval import (
+    EvaluationConfig,
+    TrainingConfig,
+    TrainLoopConfig,
+)
+from sematic.types.types.aws.s3 import S3Location
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+DEFAULT_CHECKPOINT_DIR = S3Location.from_uri("s3://sematic-dev/ray-demo")
```

Comment:
`sematic-dev` is our bucket for artifacts, logs, and blobs, i.e. framework-level objects.

Can we use a different bucket for "user-code"-generated objects?